maxscale.cnf.d/example.cnf

```diff
@@ -56,6 +54,13 @@ router=readwritesplit
 servers=server1,server2,server3
 user=maxuser
 passwd=maxpwd
+master_failure_mode=error_on_write
+
+[MaxInfo]
```

Comment:
MaxInfo isn't really recommended now that the REST API is available. Anything you can do with MaxInfo you can do more securely with the REST API.

Reply:
Removed and exposed json endpoint externally.